docs/getting_started.rst

```diff
@@ -76,18 +88,20 @@ Python virtual environment
 
 You should use a Python virtual environment to isolate the dependencies of your Python projects from each other and to avoid corrupting your system's Python installation.
 
-There are many ways to set up Python virtual environments: You can use `Pipenv <https://pipenv.readthedocs.io/en/latest/>`__ as shown in the instructions below; you can also use `Virtualenv <https://virtualenv.pypa.io/en/latest/>`__, `Python 3 venv <https://docs.python.org/3/library/venv.html>`__, `Poetry <https://poetry.eustace.io>`__ etc.
+There are many ways to set up Python virtual environments: You can use `pyenv-virtualenv <https://github.com/pyenv/pyenv-virtualenv>`__ as shown in the instructions below; you can also use `Virtualenv <https://virtualenv.pypa.io/en/latest/>`__, `Virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/>`__ `Pipenv <https://pipenv.readthedocs.io/en/latest/>`__, `Python 3 venv <https://docs.python.org/3/library/venv.html>`__, `Poetry <https://poetry.eustace.io>`__ etc.
 
 .. note::
   Most virtual environments will require special setup for non-Bash shells such as Fish and ZSH.
 
-Once Pipenv is installed, you can use the following commands to set up and use a virtual environment from within the Kolibri repo:
+To setup and start using pyenv-virtualenv, follow the instructions here :doc:`/howtos/pyenv_virtualenv`.
+
+Once pyenv-virtualenv is installed, you can use the following commands to set up and use a virtual environment from within the Kolibri repo:
 
 
 .. code-block:: bash
 
-  pipenv --python 3  # can also make a python 2 environment
-  pipenv shell  # activates the virtual environment
+  pyenv virtualenv 3.9.9 kolibri-py3.9  # can also make a python 2 environment
+  pyenv activate kolibri-py3.9  # activates the virtual environment
 
 Now, any commands you run will target your virtual environment rather than the global Python installation. To deactivate the virtualenv, simply run:
 
```

Comment:
``exit`` actually closes the terminal window, which does deactivate the virtual environment, but not in a sense we're intending here, right?. It's a bit confusing because the sentence below says _... you'll want to leave it activated..._

Reply:
Hrm, yes, it will. I'll update that.